python/runtime/utils/py_tensor.cpp

```diff
@@ -93,7 +93,7 @@ and other Python libraries that support the buffer protocol.
              py::buffer_info info = b.request();
              std::vector<std::size_t> shape(info.shape.begin(),
                                             info.shape.end());
-             t.take(static_cast<Scalar *>(info.ptr), shape);
+             //t.take(static_cast<Scalar *>(info.ptr), shape);
```

Comment:
This must be fixed. I assume it needs another std::accumulate sort of call to determine the size of the info,.ptr data.